operatorcert/utils.py

```diff
@@ -0,0 +1,164 @@
+import json
+import pathlib
+from urllib.parse import urljoin
+from typing import Any, Dict, List, Tuple
+
+import requests
+import yaml
+
+
+# Bundle annotations
+OCP_VERSIONS_ANNOTATION = "com.redhat.openshift.versions"
+PACKAGE_ANNOTATION = "operators.operatorframework.io.bundle.package.v1"
+
+# ClusterServiceVersion annotations & properties
+OLM_PROPS_ANNOTATION = "olm.properties"
+MAX_OCP_VERSION_PROPERTY = "olm.maxOpenShiftVersion"
+
+
+def find_file(base_path, relative_paths: List[Tuple[str]]) -> pathlib.Path:
+    """
+    Finds the first file that exists in a list of relative paths.
+
+    Args:
+        base_path (Path): A root path to start from
+        relative_paths (List[Tuple[str]]): An ordered list of relative paths
+            (in tuples) to look for.
+
+    Returns:
+        A Path object for the first file found; None otherwise.
+    """
+    for path in relative_paths:
+        new_path = base_path.joinpath(*path)
+        if new_path.exists() and new_path.is_file():
+            return new_path
+    return None
+
+
+def get_bundle_annotations(bundle_path: pathlib.Path) -> Dict:
```

Comment:
Could you also replace existing functions that have the same functionality?
`bundle_dockerfile` does exactly the same thing with
```
    annotation_file_path = check_annotation_file(args.bundle_path)
    annotations = read_annotation(annotation_file_path)
```

Reply:
Done.